Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Source DC Voltage.lvproj

```diff
@@ -0,0 +1,181 @@
+﻿<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Why the difference in file name for this .lvproj file? All the others are NIDCPowerSourceDCVoltage*.

Reply:
I created the example using the measurement template before I started stealing files from the python example. I will go ahead and change the project and library name to be consistent.